### PR TITLE
Turkish Translation

### PR DESCRIPTION
**Pull request by metezd**

### CODE REVIEW

**Comment by Jean-BaptisteC** [comment]
Thanks for your work,
I wait 3.0 version (Kotlin Rewrite) to approve this pull request.

**Comment by Jean-BaptisteC** [comment]
3.0 version is planned to september.
Can you update your commits with latest version of [strings.xml](https://github.com/Exodus-Privacy/exodus-android-app/blob/kotlinRewrite/app/src/main/res/values/strings.xml) ?

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="app_title" translatable="false">εxodus</string>
```
Comment (Jean-BaptisteC) [inline app/src/main/res/values-tr/strings.xml:2]: Please remove there 8 strings with attributes translatable = false

File: app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <string name="app_trackers" translatable="false"><xliff:g id="rating">%1$d</xliff:g></string>
+    <string name="app_perms" translatable="false"><xliff:g id="rating">%1$d</xliff:g></string>
+    <string name="fetching_apps">Rapor veritabanı güncelleniyor!</string>
+
+    <!-- About Fragment -->
+    <string name="version_info" translatable="false"><xliff:g id="type">%1$s</xliff:g> <xliff:g id="name">%2$s</xliff:g>-<xliff:g id="code">%3$d</xliff:g></string>
+    <string name="website">Web sitesi</string>
+    <string name="website_desc">Exodus Privacy projesi hakkında daha fazla bilgi edinin</string>
+    <string name="srcCode" translatable="false">Github</string>
+    <string name="srcCode_desc">Sorunları bildirin, yıldız verin, PR'lara katkıda bulunun veya kodu kontrol edin</string>
```
Comment (Jean-BaptisteC) [inline app/src/main/res/values-tr/strings.xml:46]:
```suggestion
    <string name="srcCode_desc">Sorunları bildirin, yıldız verin, PR\'lara katkıda bulunun veya kodu kontrol edin</string>
```

File: app/src/main/res/values-tr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    <string name="detection_rules">Tespit Kuralları</string>
+    <string name="code_signature">Kod saptama kuralı:</string>
+    <string name="network_signature">Ağ saptama kuralı:</string>
+    <string name="apps_list">Uygulama listesi</string>
+    <string name="trackers_presence">Uygulamalarınızın <xliff:g id="percentage">%1$d</xliff:g>% (<xliff:g id="apps">%2$d</xliff:g> uygulama) oranında mevcut</string>
+
+    <!-- Dialog Fragment -->
+    <string name="accept">Kabul et</string>
+    <string name="reject">Reddet</string>
+    <string name="warning_title">Exodus'un gerekli verileri iletmesine izin verilsin mi?</string>
```
Comment (Jean-BaptisteC) [inline app/src/main/res/values-tr/strings.xml:97]:
```suggestion
    <string name="warning_title">Exodus\'un gerekli verileri iletmesine izin verilsin mi?</string>
```

**Review by Jean-BaptisteC — CHANGES_REQUESTED** [review]
Thanks for translation.
Please check my comments.

**Comment by metezd** [comment]
Updated all of them.

**Review by Jean-BaptisteC — APPROVED** [review]

**Pull request merged**